Tutorial12.6/responsive-layout.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title> Responsive Layout </title>
+    <style>
+        body{
+            color: #fff;
+            font-family: 'Nunito Semibold';
+            text-align: center;
+        }
+        #content{
+            display: grid;
+            grid-template-columns: repeat(4,1fr);
+            grid-auto-rows: minmax(100px, auto);
+            grid-gap: 10px;
+            max-width: 960px;
+            margin: 0 auto;
+            grid-template-areas: 
+            "header header header header"
+            "footer footer footer footer"
+            "main main main main"
+            "main main main main"
+            "aside aside nav nav"
+            "section section section section"
+            "section section section section";
+        }
+    @media screen and (min-width:760px){ 
+        #content{
```

Comment:
768px is the standard breakpoint for tab screens

Reply:
ok